.github/workflows/backport.yaml

```diff
@@ -9,6 +9,10 @@ on:
       - closed
       - labeled
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
```

Comment:
This should be modified in https://github.com/autowarefoundation/sync-file-templates/blob/92400f065df07629befbfe48e808649ee3581f11/sources/.github/workflows/backport.yaml#L11

Reply:
Actually, because of the same reason, this shouldnt be added in the first place:
- https://github.com/autowarefoundation/sync-file-templates/pull/49